### PR TITLE
prometheus: update to 2.24.1

### PR DESCRIPTION
**Pull request by paper42**

### CODE REVIEW

**Comment by PaperMountainStudio** [comment]
xlint returns a false warning
```
srcpkgs/prometheus/template:31: license Apache-2.0 should not be installed
```
the `vlicense NOTICE` line should not be removed, because it contains licenses from third party components.

File: srcpkgs/prometheus/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'prometheus'
 pkgname=prometheus
-version=2.21.0
+version=2.24.0
```
Comment (ericonr) [inline srcpkgs/prometheus/template:3]: We should move to 2.24.1, it has an apparently important bugfix.

**Comment by ahesford** [comment]
If the notice includes third-party license terms, you should probably amend the license specifier to include those additional licenses.

**Comment by ericonr** [comment]
It can happen on the next release :)

**Pull request merged**